homeworks/vladislav.buzunar_buzyaka/3-functions/functions.js

```diff
@@ -0,0 +1,26 @@
+const OIL = 926;
+const WATER = 1000;
+const MERCURY = 1355;
+
+const numOilBank = 3;
+const numWaterBank = 5;
+const numMercuryBank = 2;
+
+function getWeight(weight, amount) {
```

Comment:
This function should take 3 arguments: `getWeight(weight, liquid, count)`

For example:
```
const TANK_VOLUME = 1000;
const OIL = 926;

const numOilBank = 3;

const totalWeightOil = getWeight(TANK_VOLUME, OIL, numOilBank);
```